src/Tizen.Multimedia/Interop/Interop.Device.cs

```diff
@@ -60,6 +60,36 @@ internal static partial class AudioDevice
             [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_is_device_running_by_id")]
             internal static extern AudioManagerError IsDeviceRunning(int deviceId, out bool isRunning);
 
+            [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_get_supported_sample_formats_by_id")]
+            internal static extern AudioManagerError GetSupportedSampleFormats(int deviceId, out IntPtr formats, out uint numberOfElements);
+
+            [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_set_sample_format_by_id")]
+            internal static extern AudioManagerError SetSampleFormat(int deviceId, AudioSampleFormat format);
+
+            [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_get_sample_format_by_id")]
+            internal static extern AudioManagerError GetSampleFormat(int deviceId, out AudioSampleFormat format);
+
+            [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_get_supported_sample_rates_by_id")]
+            internal static extern AudioManagerError GetSupportedSampleRates(int deviceId, out IntPtr rates, out uint numberOfElements);
+
+            [DllImport(Libraries.SoundManager, EntryPoint = "sound_manager_set_sample_rate_by_id")]
```

Comment:
Why don't you use ENUM like Native FW? ```rate``` parameter is ```sound_sample_rate_e``` enum.
Is there any reason for this? 
I think, if you introduce it at C# layer, you don't need to covert it at all.

```
typedef enum {
	SOUND_SAMPLE_RATE_8000,  
	...
	SOUND_SAMPLE_RATE_192000,
} sound_sample_rate_e;
```

Reply:
We need to think that the change like Native APIs is really useful for C# user from the C# API point of view.

Reply:
I think enum value is more useful and help to prevent misuse(intellisense help to choice it).  Actually AudioIO API use sample rate as int, but there's no resctition for it. But your API doesn't allowed another sample rate expect you predefined.

Reply:
It differs from audio-io situation. In case of audio-io, s/w resampler is in charge of resampling, so it's not that strict. In contrast, in case of use device, these are from supported values of h/w device(of course via ALSA). So if user set any value without quiring supported rates from API, it will occurs an error. This situation is also same even if we make it enum(intellisense does not know the supported sample rates anyway).